### PR TITLE
Setings page logo adjustments

### PR DESCRIPTION
**Pull request by Samq64**
### Changes

Removes the white space from `icon-transparent.svg` and makes it appear bigger in header, but not as big as #5104. The button hover from #7333 has also been added to the logo on the license and permission pages.

### Reason for changes

The hover is nice since it's otherwise not obvious the logo is clickable and the SVG white space only exists because I was lazy in #4810.

### Tests

Tested in Chromium and Firefox. RTL also seems fine.

### CODE REVIEW

**Comment by Waakul** [comment]
I would recommend that you provide screenshots

**Comment by Samq64** [comment]
| Before | After |
|-----|-----|
|popup-old | popup-new |
| settings-old | settings-new
| settings-small-old | settings-small-new
| other-pages-old | other-pages-new |

**Review by DNin01 — COMMENTED** [review]
For the pages, I like how the logo is bigger, but I did like the slightly more space between the text, the logo, and the beginning/left edge of the page from before.

I do like how it looks on the popup after this change, though!

**Review by mxmou — CHANGES_REQUESTED** [review]

File: webpages/settings/index.html [hunk 1]
```diff
@@ -120,10 +120,10 @@
   </head>
   <body v-cloak :class="{light: theme}">
     <div class="navbar">
-      <img :src="switchPath" class="toggle" @click="sidebarToggle()" v-cloak v-show="smallMode" alt="Logo" />
+      <img :src="switchPath" class="header-button" @click="sidebarToggle()" v-cloak v-show="smallMode" alt="Logo" />
```
[the remark this answers is not in the document]
Comment (Samq64) [reply webpages/settings/index.html:123]: If the styles aren't necessary could it be changed to an ID?

**Comment by Samq64** [comment]
> For the pages, I like how the logo is bigger, but I did like the slightly more space between the text, the logo, and the beginning/left edge of the page from before.

That's a side effect of the image not being square anymore.

**Review by mxmou — CHANGES_REQUESTED** [review]
Bug: the icon isn't centered on the Licenses page. This is easiest to see when hovered:
image
It could be fixed by adding either `display: flex` or `line-height: 0` to the `<a>` element.

The spacing on the settings page is now better, but I think it should be the same as in the popup (20px on both sides). Ideally, the spacing on the Licenses page would also be the same (`padding: 17.5px 20px` and no margin).

**Comment by Samq64** [comment]
The logo on the permission page is also a link. Does that make sense to keep?

**Comment by mxmou** [comment]
> The logo on the permission page is also a link. Does that make sense to keep?

It's not very useful, but I don't think it needs to be removed.

**Review by mxmou — APPROVED** [review]

**Comment by WorldLanguages** [comment]
@Samq64 No screenshots?

**Comment by Samq64** [comment]
https://github.com/ScratchAddons/ScratchAddons/pull/7485#issuecomment-2143444073

Things changed a little after Mxmou's suggestions

**Comment by WorldLanguages** [comment]
Okay, you both are org members so I'm merging this,

**Pull request merged**